common/src/main/java/com/microsoft/identity/common/adal/internal/tokensharing/TokenShareUtility.java

```diff
@@ -180,6 +207,57 @@ public void saveFamilyRefreshToken(@NonNull final String tokenCacheItemJson) thr
         }
     }
 
+    @Override
+    public String getMsaFamilyRefreshToken(@NonNull final String identifier) throws Exception {
+        final ICacheRecord cacheRecord = getCacheRecordForIdentifier(identifier);
+
+        throwIfCacheRecordIncomplete(identifier, cacheRecord);
+
+        return cacheRecord.getRefreshToken().getSecret();
```

Comment:
Is cacheRecord.getRefreshToken() always nonnull?

Reply:
Yep - `throwIfCacheRecordIncomplete` will check the contents of the cache record and throw an Exception if it's incomplete